src/node/internal/crypto_keys.ts

```diff
@@ -190,6 +210,10 @@ export abstract class KeyObject {
   }
 }
 
+export function isKeyObject(obj: any): obj is KeyObject {
```

Comment:
```suggestion
export function isKeyObject(obj: any): asserts obj is KeyObject {
```

Reply:
Given that the function returns boolean, adding the asserts doesn't work.